docs/settings.rst

```diff
@@ -240,3 +235,23 @@ Default: ``False``
    GRAPHENE = {
       'GRAPHIQL_SHOULD_PERSIST_HEADERS': False,
    }
+
+
+``GRAPHIQL_INPUT_VALUE_DEPRECATION``
+------------------------------------
+
+Set to ``True`` if you want GraphiQL to display a deprecation section on field doc view.
+
+This setting is passed to ``inputValueDeprecation`` GraphiQL options, for details refer to GraphiQLDocs_.
+Make sure your GraphQL server supports deprecation of input values before setting this to ``True``.
```

Comment:
I'm confused by this comment. Is this referring to the version of `graphql-core` (https://github.com/graphql-python/graphql-core), or `graphene`, or something else? In what situation would it not be supported?

Reply:
This sentence is actually wrong. I meant checking `graphql-js` version.

Where we could have a risk is when we would use a version of GraphiQL that would depend on an old version of `graphql` (i.e. `graphql-js`) that does not support input field deprecation, see: https://github.com/graphql/graphiql/blob/57b5fcd8749bd1c54f4a6a04a9ab07316fd05e71/packages/graphiql/resources/renderExample.js#L103

Only `graphql>15.5` supports input field deprecation.

After checking, it seems anyway that graphene-django pins `graphiql`'s version here: https://github.com/graphql-python/graphene-django/blob/main/graphene_django/views.py#L68 to `2.4.7` and this version of `graphiql` in turn uses `graphql@16.5.0`: https://github.com/graphql/graphiql/blob/graphiql%402.4.7/yarn.lock#L9991 which is above `15.5`.

So I think I can just remove this warning as it should no longer be a problem.